go.mod

```diff
@@ -90,81 +109,92 @@ require (
+	github.com/petermattis/goid v0.0.0-20230518223814-80aa455d8761 // indirect
 	github.com/pkg/errors v0.9.1 // indirect
 	github.com/pmezard/go-difflib v1.0.0 // indirect
-	github.com/prometheus/client_golang v1.14.0 // indirect
-	github.com/prometheus/client_model v0.3.0 // indirect
-	github.com/prometheus/common v0.42.0 // indirect
-	github.com/prometheus/procfs v0.9.0 // indirect
+	github.com/prometheus/client_golang v1.16.0 // indirect
+	github.com/prometheus/client_model v0.4.0 // indirect
+	github.com/prometheus/common v0.44.0 // indirect
+	github.com/prometheus/procfs v0.10.1 // indirect
 	github.com/rcrowley/go-metrics v0.0.0-20201227073835-cf1acfcdf475 // indirect
+	github.com/rogpeppe/go-internal v1.10.0 // indirect
 	github.com/rs/cors v1.8.3 // indirect
 	github.com/rs/zerolog v1.29.1 // indirect
 	github.com/sasha-s/go-deadlock v0.3.1 // indirect
 	github.com/spf13/afero v1.9.5 // indirect
 	github.com/spf13/jwalterweatherman v1.1.0 // indirect
-	github.com/spf13/pflag v1.0.5 // indirect
-	github.com/spf13/viper v1.16.0 // indirect
 	github.com/subosito/gotenv v1.4.2 // indirect
 	github.com/syndtr/goleveldb v1.0.1-0.20220721030215-126854af5e6d // indirect
 	github.com/tendermint/go-amino v0.16.0 // indirect
 	github.com/tidwall/btree v1.6.0 // indirect
-	github.com/ugorji/go/codec v1.2.7 // indirect
 	github.com/ulikunitz/xz v0.5.11 // indirect
 	github.com/zondax/hid v0.9.1 // indirect
 	github.com/zondax/ledger-go v0.14.1 // indirect
 	go.etcd.io/bbolt v1.3.7 // indirect
 	go.opencensus.io v0.24.0 // indirect
-	golang.org/x/crypto v0.9.0 // indirect
-	golang.org/x/exp v0.0.0-20230515195305-f3d0a9c9a5cc // indirect
+	golang.org/x/crypto v0.10.0 // indirect
+	golang.org/x/exp v0.0.0-20230522175609-2e198f4a06a1 // indirect
 	golang.org/x/net v0.10.0 // indirect
-	golang.org/x/oauth2 v0.7.0 // indirect
-	golang.org/x/sys v0.8.0 // indirect
-	golang.org/x/term v0.8.0 // indirect
-	golang.org/x/text v0.9.0 // indirect
+	golang.org/x/oauth2 v0.8.0 // indirect
+	golang.org/x/sync v0.3.0 // indirect
+	golang.org/x/sys v0.9.0 // indirect
+	golang.org/x/term v0.9.0 // indirect
+	golang.org/x/text v0.10.0 // indirect
 	golang.org/x/xerrors v0.0.0-20220907171357-04be3eba64a2 // indirect
 	google.golang.org/api v0.122.0 // indirect
 	google.golang.org/appengine v1.6.7 // indirect
+	google.golang.org/genproto v0.0.0-20230525234025-438c736192d0 // indirect
+	google.golang.org/genproto/googleapis/rpc v0.0.0-20230530153820-e85fd2cbaebc // indirect
 	gopkg.in/ini.v1 v1.67.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
+	gotest.tools/v3 v3.4.0 // indirect
 	nhooyr.io/websocket v1.8.6 // indirect
-	pgregory.net/rapid v0.5.5 // indirect
+	pgregory.net/rapid v1.0.0 // indirect
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
 replace github.com/syndtr/goleveldb => github.com/syndtr/goleveldb v1.0.1-0.20210819022825-2ae1ddf74ef7
+
+replace github.com/cosmos/ibc-go/modules/capability => ./modules/capability
```

Comment:
Maybe we should pin to a version of the capability module? cc @damiannolan @chatton

Reply:
yeah we should be able to pin this, it looks like on main we are pinning to `v1.0.0-rc1`, I see that in the capability module is now also pointing at the local copy of ibc-go. It sounds like there are some issues importing modules on an older version of the sdk?

Reply:
I think it could potentially make things easier to upgrade the capability module to eden separately

Reply:
@damiannolan  you mean that keeping a local pin will make the upgrade easier as a followup?

Reply:
@damiannolan had the same thought, and good news! this is one piece that is completed.  Let me break it out....

Reply:
@damiannolan it seems that the capability module has dependencies inside of the rest of ibc-go:

```log
go test ./...
# github.com/cosmos/ibc-go/v7/modules/apps/27-interchain-accounts/host/client/cli
../apps/27-interchain-accounts/host/client/cli/query.go:85:37: r.Events undefined (type "github.com/cosmos/cosmos-sdk/types".SearchTxsResult has no field or method Events)
../apps/27-interchain-accounts/host/client/cli/query.go:88:53: cannot use resEvents (variable of type []"cosmossdk.io/api/tendermint/abci".Event) as []"github.com/cometbft/cometbft/abci/types".Event value in argument to sdk.StringifyEvents
../apps/27-interchain-accounts/host/client/cli/query.go:99:24: undefined: searchEvents
../apps/27-interchain-accounts/host/client/cli/query.go:100:38: undefined: clientCtx
../apps/27-interchain-accounts/host/client/cli/query.go:104:29: cannot use result (variable of type *"github.com/cosmos/cosmos-sdk/types".SearchTxsResult) as "github.com/cosmos/cosmos-sdk/types".SearchTxsResult value in argument to append
FAIL    github.com/cosmos/ibc-go/modules/capability [build failed]
ok      github.com/cosmos/ibc-go/modules/capability/keeper      0.613s
FAIL    github.com/cosmos/ibc-go/modules/capability/simulation [build failed]
ok      github.com/cosmos/ibc-go/modules/capability/types       1.010s
FAIL
```

Reply:
@chatton I can... try something maybe it will work?